docs/developer-docs/integrations/sns/managing-sns/sns-asset-canister.md

```diff
@@ -0,0 +1,162 @@
+# SNS asset canisters
+
+## Overview
+
+The [asset canister](https://github.com/dfinity/sdk/tree/master/src/canisters/frontend/ic-frontend-canister) provides users with a way to store and retrieve static assets from a canister deployed on the IC. Generally, asset canisters are used to serve HTML, CSS, or JavaScript assets, which are typically part of an application's frontend. For this reason, the asset canister is also referred to as the frontend canister. For purposes of this guide, we'll refer to it as the asset canister. 
+
+The asset canister of an SNS serves the frontend assets related to the SNS. The contents of the asset canister must be configured prior to the launch of the SNS, and any changes afterwards must be approved on via an SNS proposal. These changes are referred to as 'upgrades' to the asset canister in the remainder of this document. 
+
+This section is relevant if your project contains an asset canister and describes how you can test handing over control of an asset canister to an SNS.
+
+:::info
+Note that you can interleave the steps for the asset canister from this section and the steps for the test canister (and also your own canister) within the SNS lifecycle. We list additional constraints in between the steps below.
```

Comment:
I am not sure if it is clear how this page relates to all other pages..
Do we describe testing here? or what to do in production? or both?